io/net/virt-net/lpm.py.data/README.txt

```diff
@@ -0,0 +1,34 @@
+Install Paramiko module:
+Ubuntu :
+-> apt-get install python-paramiko
+-SLES/RHEL :
+SLES :
+
+-> SLES: Install python-devel,python-setuptools
+--> RHEL: Install python-devel.ppc64le and python-setuptools.noarch
+-> Download pycrypto-2.6.1.tar.gz & paramiko-1.16.0.tar.gz
+-> For pycrypto : python setup.py build & python setup.py install
+-> For parmaiko : easy_install ./
+-> Then import paramiko.
```

Comment:
i did not see paramiko used any where in test

Reply:
My bad. Forgot to remove this.